tariff-management-service/app/src/main/java/com/microservices/tariffmanagement/service/TariffServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.microservices.tariffmanagement.service;
+
+import com.microservices.tariffmanagement.dto.TariffDto;
+import com.microservices.tariffmanagement.entity.TariffEntity;
+import com.microservices.tariffmanagement.repository.TariffRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TariffServiceImpl implements TariffService {
+    private final Logger logger = LoggerFactory.getLogger(TariffServiceImpl.class);
+    private TariffRepository tariffRepository;
+
+    @Autowired
+    public TariffServiceImpl(TariffRepository tariffRepository) {
+        this.tariffRepository = tariffRepository;
+    }
+
+    @Override
+    public long createTariff(TariffDto tariff) {
+        logger.info("Creating tariff: {}...", tariff);
+        TariffEntity tariffEntity = new TariffEntity(tariff);
+        int id = tariffRepository.save(tariffEntity).getId();
+        logger.info("Created new tariff with id {}", id);
+        return id;
```

Comment:
we should return representational state of tariff

Reply:
You mean logging? Or returning the object? In PendingDetailServiceImpl you are returning just the id

Reply:
I mean returning the object.  We behave this way in Pending details because we just return the value for printing barcode.